### PR TITLE
Update focal loss definition in focal_loss docs

### PR DESCRIPTION
**Pull request by Anirban0011**
## What this PR does:

* Attempts to expand upon the existing definition of focal loss in the docstring.

### CODE REVIEW

**Comment by Anirban0011** [comment]
@ianstenbit @LukeWood , kindly review whenever possible . Thanks!

File: keras_cv/losses/focal.py [hunk 1]
```diff
@@ -22,7 +22,11 @@ class FocalLoss(keras.losses.Loss):
     """Implements Focal loss
 
     Focal loss is a modified cross-entropy designed to perform better with
-    class imbalance. For this reason, it's commonly used with object detectors.
+    class imbalance. It minimizes the weight of easy -ves on the loss function
```
Comment (ianstenbit) [inline keras_cv/losses/focal.py:25]: Since we already reference the FocalLoss paper below, I think this detail is a bit extraneous (it's not critical information to a user of this API, and if they want to learn more they're already referred to the paper)

**Comment by ianstenbit** [comment]
Thanks for the PR! See my comment above -- I think we should leave this doc as-is for now.
